File: src/Wishes.java
public enum Wishes {
    SOUP ("Требую на обед варили суп из конфет и кашу из мармелада"),
    CAULDRON ("Заказываю котлеты из земляники с грибным соусом"),
    PUREE ("Приказываю принести яблочное пюре");

    private String title;

    Wishes(String title) {
        this.title = title;
    }
    public String getTitle() {
        return title;
    }

    @Override
    public String toString() {
        return "Wishes{" +
                "title='" + title + '\'' +
                '}';
    }
}

File: kotlin_fans/Wishes.kt
enum class Wishes(val title: String) {
    SOUP("Требую на обед варили суп из конфет и кашу из мармелада"),
    CAULDRON("Заказываю котлеты из земляники с грибным соусом"),
    PUREE("Приказываю принести яблочное пюре");

    override fun toString(): String {
        return "Wishes{" +
                "title='" + title + '\'' +
                '}'
    }
}